versionedRouter.js

```diff
@@ -18,10 +19,10 @@ const {
   isHttpStatusSuccess,
   getErrorRespEvents,
   isCdkDestination,
-  getErrorStatusCode
-} = require("./v0/util");
+  getErrorStatusCode,
+  getDestHandler
+} = v0Utils;
```

Comment:
Rather than destructuring, directly use the utils object to invoke the methods.

Reply:
We don't want to make changes in other parts unless it is absolutely necessary

Reply:
Let's not have two different ways of accessing the utils in the versioned router, at least. It sets a bad precedent for future developers.